libraries/RTC/src/RTC.h

```diff
@@ -8,24 +8,39 @@ class APM3_RTC
 public:
   APM3_RTC();
 
-  void getTime(); //Query the RTC for the current time/date. Loads .seconds, .minute, etc.
-  void setTime(uint8_t hour, uint8_t min, uint8_t sec, uint8_t hund,
-               uint8_t dayOfMonth, uint8_t month, uint16_t year); //Set current time to provided hundredths/seconds/etc
-  void setToCompilerTime();                                       //Set to time when sketch was compiled
+  void getTime(); //Query the RTC for the current time/date
+  uint32_t getEpoch(); //Return the current RTC time/date as UNIX Epoch time 
+  
+  void setTime(uint8_t hund, uint8_t sec, uint8_t min, uint8_t hour, uint8_t dayOfMonth, uint8_t month, uint16_t year); //Set current time to provided hundredths/seconds/etc
```

Comment:
Reordering the setTime parameters is a breaking API change. Would prefer this change to occur as part of transition to v2.0.0 while still getting the functional improvements brought into the v1.x.x lineage